pylama.ini

```diff
@@ -27,3 +27,6 @@ complexity = 10
 [pylama:pylint]
 good-names = logger,e,application,gunicorn_logger
 ignored-classes = logger
+
+[pylama:volume_type_validation/volume_type_validation.py]
+ignore = E501,C0301,R0201
```

Comment:
why ignoring these?
can't the offending places be fixed?

Reply:
The long line ones can be fixed, but should be done in a separate PR.

To give you some background, this was our very first use case, and mostly taken as is from the notebook implementation in gitlab.
The focus back then was to just get it integrated with the pipeline, and hence the long lines were left untouched.

Reply:
I had to ignore those for now, for pytest to run the tests in the current PR

Reply:
got it